grails-app/controllers/com/unifina/controller/api/StreamApiController.groovy

```diff
@@ -66,6 +66,9 @@ class StreamApiController {
 			stream.name = newStream.name
 			stream.description = newStream.description
 			stream.config = readConfig()
+			stream.autoConfigure = newStream.autoConfigure
```

Comment:
Should we only overwrite these fields if they are present? Otherwise we risk overwriting them with `null`
```suggestion
			stream.autoConfigure = newStream.autoConfigure != null ? newStream.autoConfigure : stream.autoConfigure
```

Reply:
For some reason I tend to forget about 'zero value'.